dev/tasks/tasks.yml

```diff
@@ -447,3 +449,8 @@ tasks:
       - parquet-glib-doc-{no_rc_version}-1.el7.x86_64.rpm
       - parquet-glib-libs-{no_rc_version}-1.el7.x86_64.rpm
       - parquet-libs-{no_rc_version}-1.el7.x86_64.rpm
+
+  ############################## Gandiva Tasks ####################################
+  ubuntu-trusty-gandiva:
+    platform: linux
+    template: gandiva/travis.linux.yml      
```

Comment:
It requires a tasks key as well, see for example https://github.com/apache/arrow/blob/master/dev/tasks/tasks.yml#L420

Reply:
I am sorry i could not follow, i've added a group and a task definition.
do you mean that i should call out the artifacts too?

Reply:
We have a status command which checks whether the expected artifacts are uploaded as github release assets or not.